config/install/core.entity_form_display.node.event.default.yml

```diff
@@ -66,6 +68,7 @@ third_party_settings:
       region: content
     group_right_column:
       children:
+        - group__related_links
```

Comment:
@enzolutions Could you please update this to be group_related_links (only 1 underscore between group and related_links)